src/OpenLoco/src/Objects/SnowObject.cpp

```diff
@@ -13,11 +16,15 @@ namespace OpenLoco
     // 0x00469A35
     void SnowObject::load(const LoadedObjectHandle& handle, stdx::span<const std::byte> data, ObjectManager::DependentObjects*)
     {
-        Interop::registers regs;
-        regs.esi = Interop::X86Pointer(this);
-        regs.ebx = handle.id;
-        regs.ecx = enumValue(handle.type);
-        Interop::call(0x00469A35, regs);
+        auto remainingData = data.subspan(sizeof(SnowObject));
+
+        auto strRes = ObjectManager::loadStringTable(remainingData, handle, 0);
+        name = strRes.str;
+        remainingData = remainingData.subspan(strRes.tableLength);
+
+        auto imgRes = ObjectManager::loadImageTable(remainingData);
+        image = imgRes.imageOffset;
```

Comment:
are you sure this is correct there looks to be a + 95 in the assembly

Reply:
I've only done a visual inspection. Seems fine to me, but I'll check again tomorrow.

Reply:
Looks like offset 95 is where the tiles at full zoom level start, so this was likely a very intentional offset. I'll add some constants for each of the zoom levels.